src/components/MultiGestureCanvas/getCanvasFitScale.ts

```diff
@@ -0,0 +1,22 @@
+type GetCanvasFitScale = (props: {
+    canvasSize: {
+        width: number;
+        height: number;
+    };
+    contentSize: {
+        width: number;
+        height: number;
+    };
+}) => {scaleX: number; scaleY: number; minScale: number; maxScale: number};
+
+const getCanvasFitScale: GetCanvasFitScale = ({canvasSize, contentSize}) => {
```

Comment:
👋 Coming from https://github.com/Expensify/App/issues/40788
This has introduced an issue on native Android, where if you're viewing a small image, it's scaled up to fill all of the screen
We've resolved this by capping the `scaleX` and `scaleY` at 1, so the images are no longer scaled up automatically

Reply:
makes sense! thanks for the catch 🙌